### DISCUSSION TITLE
Template Format

### DISCUSSION THREAD

**Discussion by BossJake** [rfennell/AzurePipelines#1014] [General]
Hi There,

How can I achieve this release notes format same as the link below using the cross platform extension? 
Basically, I want to create a Release Notes definition task in Azure Devops where I can define the content of a specific release.

- description 
- features
- associated work items

[https://imgur.com/a/UyOvZe7](url)

**Comment by rfennell** [comment]
The link you provided does not work. 

However, all the information you need can be found in the extensive documentation I have written for the task in the [wiki](https://github.com/rfennell/AzurePipelines/wiki/GenerateReleaseNotes---Node-based-Cross-Platform-Task). There are also more template samples [here](https://github.com/rfennell/AzurePipelines/tree/main/SampleTemplates/XplatGenerateReleaseNotes%20(Node%20based)/Version%203).

Also, you might find the command test harness useful for creating templates, which can be found [here](https://github.com/rfennell/AzurePipelines/tree/main/Extensions/XplatGenerateReleaseNotes/V3/testconsole)

**Reply by BossJake** [reply]
Thanks for your reply. I'm looking into those templates that you mentioned. 

Here's the link that is working.

[here](https://imgur.com/uFyBT9O)

**Comment by rfennell** [comment]
From a quick look at the sample, seems what you want is possible.

- Start with basic loops on the `workitems` and `commits` to get items since the last successful build
- To get `feature` look at enabling parent workitems, or filter the `workitem` array based on WIT
